test/metabase/models/permissions_test.clj

```diff
@@ -620,6 +620,15 @@
                                :query :segmented}}
              (test-data-graph group))))))
 
+(deftest audit-db-update-test
+  (testing "Throws exception when we attempt to change the audit db permission manually."
+    (mt/with-temp* [PermissionsGroup [group]
+                    Database         [database]
+                    Table            [table    {:db_id (u/the-id database)}]]
+      (with-redefs [perms/default-audit-db-id (constantly (u/the-id database))]
+        (is (thrown? Exception
```

Comment:
nit: slightly nicer to use `thrown-with-msg?` and also assert on the error message